src/main/java/com/cloudogu/scm/commitmessagechecker/cli/CommitMessageCheckerCommand.java

```diff
@@ -0,0 +1,116 @@
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+package com.cloudogu.scm.commitmessagechecker.cli;
+
+import com.cloudogu.scm.commitmessagechecker.AvailableValidators;
+import com.cloudogu.scm.commitmessagechecker.Context;
+import com.cloudogu.scm.commitmessagechecker.Validator;
+import com.cloudogu.scm.commitmessagechecker.config.Configuration;
+import com.cloudogu.scm.commitmessagechecker.config.ConfigurationProvider;
+import com.cloudogu.scm.commitmessagechecker.config.Validation;
+import com.google.common.annotations.VisibleForTesting;
+import picocli.CommandLine;
+import sonia.scm.cli.CliContext;
+import sonia.scm.cli.CliResourceBundle;
+import sonia.scm.cli.ParentCommand;
+import sonia.scm.cli.TemplateRenderer;
+import sonia.scm.repository.NamespaceAndName;
+import sonia.scm.repository.Repository;
+import sonia.scm.repository.RepositoryManager;
+import sonia.scm.repository.cli.RepositoryCommand;
+
+import javax.inject.Inject;
+import java.util.List;
+
+@ParentCommand(RepositoryCommand.class)
+@CliResourceBundle("sonia.scm.cmc.cli.i18n")
+@CommandLine.Command(name = "commit-message-check", aliases = "cmc")
+public class CommitMessageCheckerCommand implements Runnable {
+
+  @CommandLine.Parameters(paramLabel = "namespace/name", descriptionKey = "scm.repo.cmc.repository", index = "0")
+  private String repository;
+
+  @CommandLine.Parameters(paramLabel = "branch", descriptionKey = "scm.repo.cmc.branch", index = "1")
+  private String branch;
+
+  @CommandLine.Parameters(paramLabel = "commit-message", descriptionKey = "scm.repo.cmc.commitMessage", index = "2")
+  private String commitMessage;
+
+  @CommandLine.Mixin
+  private final CommitMessageCheckerTemplateRenderer templateRenderer;
+  private final CliContext context;
+
+  private final ConfigurationProvider configurationProvider;
+  private final AvailableValidators availableValidators;
+  private final RepositoryManager repositoryManager;
+
+  @Inject
+  public CommitMessageCheckerCommand(
+    CommitMessageCheckerTemplateRenderer templateRenderer,
+    CliContext context,
+    ConfigurationProvider configurationProvider,
+    AvailableValidators availableValidators,
+    RepositoryManager repositoryManager
+  ) {
+    this.templateRenderer = templateRenderer;
+    this.context = context;
+    this.configurationProvider = configurationProvider;
+    this.availableValidators = availableValidators;
+    this.repositoryManager = repositoryManager;
+  }
+
+  @Override
+  public void run() {
+    String[] splitRepo = repository.split("/");
+    Repository repo = repositoryManager.get(new NamespaceAndName(splitRepo[0], splitRepo[1]));
+    configurationProvider.evaluateConfiguration(repo)
+      .ifPresent(
+        configuration -> {
+          try {
+            validate(configuration, repo);
+          } catch (RuntimeException e) {
```

Comment:
Shouldn't we check for a concrete `InvalidCommitMessageException` here?
```suggestion
          } catch (InvalidCommitMessageException e) {
```